src/config/dark-sites.config

```diff
@@ -36,6 +36,7 @@ anon.sx
 aonprd.com
 app.destinyitemmanager.com
 app.keeweb.info
+app.koyeb.com
```

Comment:
![image](https://user-images.githubusercontent.com/25481501/148582735-e8055ecd-d9fa-43d2-8a62-da9e71362717.png)
Not dark on the login page.

Reply:
Removed. side element didn't appeared on mobile view 😅